packages/mroonga-package-task.rb

```diff
@@ -169,71 +157,26 @@ def yum_expand_variable(key)
     end
   end
 
-  def define_windows_upload_task
-    namespace :windows do
-      desc "Upload packages"
-      task :upload do
-        mroonga_repository = "mroonga/mroonga"
-        tag_name = "v#{@version}"
-
-        github_token = env_value("GITHUB_TOKEN")
-        client = Octokit::Client.new(:access_token => github_token)
-        client.auto_paginate = true
-
-        appveyor_url = "https://ci.appveyor.com/"
-        appveyor_info = nil
-        client.statuses(mroonga_repository, tag_name).each do |status|
-          next unless status.target_url.start_with?(appveyor_url)
-          case status.state
-          when "success"
-            match_data = /\/([^\/]+?)\/([^\/]+?)\/builds\/(\d+)\z/.match(status.target_url)
-            appveyor_info = {
-              account: match_data[1],
-              project: match_data[2],
-              build_id: match_data[3],
-            }
-            break
-          when "pending"
-            # Ignore
-          else
-            message = "Appveyor build isn't succeesed: #{status.state}\n"
-            message << " #{status.target_url}"
-            raise message
-          end
-        end
-        if appveyor_info.nil?
-          raise "No Appveyor build"
-        end
+  def use_built_package?
+    true
+  end
 
-        releases = client.releases(mroonga_repository)
-        current_release = releases.find do |release|
-          release.tag_name == tag_name
-        end
-        current_release ||= client.create_release(mroonga_repository, tag_name)
-
-        start_build = appveyor_info[:build_id].to_i + 1
-        build_history = Veyor.project_history(account: appveyor_info[:account],
-                                              project: appveyor_info[:project],
-                                              start_build: start_build,
-                                              limit: 1)
-        build_version = build_history["builds"][0]["buildNumber"]
-        project = Veyor.project(account: appveyor_info[:account],
-                                project: appveyor_info[:project],
-                                version: build_version)
-        project["build"]["jobs"].each do |job|
-          job_id = job["jobId"]
-          artifacts = Veyor.build_artifacts(job_id: job_id)
-          artifacts.each do |artifact|
-            file_name = artifact["fileName"]
-            url = "#{appveyor_url}api/buildjobs/#{job_id}/artifacts/#{file_name}"
-            sh("curl", "--location", "--output", file_name, url)
-            options = {
-              :content_type => "application/zip",
-            }
-            client.upload_asset(current_release.url, file_name, options)
-            rm(file_name)
-          end
-        end
+  def built_package_url(target_namespace, target)
+    url = "https://github.com/mroonga/mroonga/releases/download/v#{@version}/"
+    url << "packages-#{@mysql_package}-#{target}.zip"
+  end
+
+  def download_packages(target_namespace)
```

Comment:
If we use the same format as Groonga, we can reuse existing `download_packages` defined in `packages-groonga-org-package-task.rb`?